app/views/user/contribute/_finished_tracks.erb

```diff
@@ -0,0 +1,3 @@
+<li>
```

Comment:
I don't think the partial helps. Sometimes a partial helps if we have a complicated thing that we're sending in as a local variable, and can use a simpler name, but here `track` is translated to `track`. It would be simpler to keep it in one place.